### PR TITLE
feat(tracer): propagate identifier flag

### PR DESCRIPTION
**Pull request by henprtz**

### CODE REVIEW

**Comment by ranrib** [comment]
Let's use a less scary and more meaningful term for this please - something like "ADD_ID_TO_RESPONSE"

File: epsagon/constants.py [hunk 1]
```diff
@@ -56,3 +56,4 @@ def is_strong_key(key):
 
 
 STEP_DICT_NAME = 'Epsagon'
+TRACE_ID_DICT_NAME = '_epsagon_trace_id'
```
Comment (ronyis) [inline epsagon/constants.py]: This name is not good. It's no the trace ID and it's not a dict.
Maybe just call it `epsagon_event_id`?

File: epsagon/utils.py [hunk 2]
```diff
@@ -109,6 +110,8 @@ def init(
       collection.
     :param keys_to_ignore: List of keys to ignore while extracting metadata.
     :param ignored_endpoints: List of ignored endpoints for web frameworks.
+    :param split_on_send: Split the trace on send flag
+    :param propagate_id: Inject identifiers via return value flag
```
Comment (ronyis) [inline epsagon/utils.py]: It's relevant only for Lambda, and I don't see that it gonna change, maybe update the name? (Also the env var name?)

File: epsagon/trace.py [hunk 3]
```diff
@@ -108,6 +109,7 @@ def initialize(
             keys_to_ignore,
             transport,
             split_on_send,
+            propagate_id,
```
Comment (ronyis) [inline epsagon/trace.py]: Update doc?

File: tests/wrappers/test_lambda_wrapper.py [hunk 4]
```diff
@@ -561,3 +561,84 @@ def wrapped_lambda(event, context):
...
+    side_effect=['trigger']
+)
+def test_propagate_id_to_dict_sanity(
+        trigger_factory_mock,
+        _,
+        set_exception_mock
+):
+    retval = {
+        'hello': 2,
+        '_epsagon_trace_id': 'test_request_id'
```
Comment (ronyis) [inline tests/wrappers/test_lambda_wrapper.py]: Better use const here

**Pull request merged**